.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules/
 lib/
 npm-debug.log
+package-lock.json
```

Comment:
This isn't right but I'll fix it later.

Reply:
Thank you for telling me, clearly I don't know enough about npm